src/json.ts

```diff
@@ -67,7 +83,27 @@ export const JsonStruct: Struct<Json> = union([
  * @returns Whether the value is a valid {@link Json} value.
  */
 export function isValidJson(value: unknown): value is Json {
-  return is(value, JsonStruct);
+  try {
+    getValidatedAndSanitizedJson(value as Json);
```

Comment:
Is this cast necessary? 
```suggestion
    getValidatedAndSanitizedJson(value);
```

Reply:
It's not required anymore. Now removed.